src/librustc/lint/mod.rs

```diff
@@ -89,7 +89,6 @@ macro_rules! lint_initializer {
 /// Declare a static item of type `&'static Lint`.
 #[macro_export]
 macro_rules! declare_lint {
-    // FIXME(#14660): deduplicate
```

Comment:
I think this one can stay? Intention seems to be to reduce the duplication in the two cases.